### ISSUE TITLE
Sometimes wrong name is shown for a person.

### ISSUE THREAD

**Issue by Shalini201** [estebanuri/face_recognition#34]
Hi Sir,
Sometimes the recognition is wrong, for example personA is recognized as personC. Is there any cutoff score of percentage confidence or probability, so that we can show the name only when its confidence is lets say "more than 90%" ?

**Comment by ersaurabh101** [comment]
Yes, facing same problem,  why the author never answered ? The article he wrote seem to be very professional. Need paid help for some customization on this code, if anyone is interested please contact on whatsapp +919216142737
